packages/scripts/src/local/getGasEthHistoricalPrice.js

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+
+/**
+ * @notice This is an example script demonstrating how to get a historical median gas price at a specific timestamp.
+ * It could be used for example to query a GASETH-TWAP-1Mx1M price before committing a vote for a DVM price request.
+ *
+ * @dev Prequisites: before running, you will need to set a GOOGLE_APPLICATIONS_CREDENTIALS environment variable for a service account.
+ * @dev This service account will need GCP admin or BigQuery permissions. This guide provides further instructions: https://cloud.google.com/docs/authentication/getting-started
+ * @dev How to run: ./scripts/local/getGasEthHistoricalPrice.js --time 1612137600 --identifier GASETH-TWAP-1Mx1M
+ */
+const { BigQuery } = require("@google-cloud/bigquery");
+const highland = require("highland");
+const moment = require("moment");
+const Web3 = require("web3");
+const { fromWei } = Web3.utils;
+const argv = require("minimist")(process.argv.slice(), { string: ["identifier", "time"] });
+const { createQuery } = require("./GasEthQuery");
```

Comment:
This depends on the separate `GasEthQuery` which is no longer in the same directory.

Reply:
Should be fixed!